### ISSUE TITLE
Source map issues

### ISSUE THREAD

**Issue by thorn0** [squidfunk/karma-viewport#509]
#458 doesn't look like the correct solution:

```diff
  pattern: file,
  included: true,
  served,
  watched: false
```

`served: false` means that the request to this file will lead to 404, and indeed that's what happens for me.

```
13 10 2020 15:25:17.700:WARN [middleware:karma]: Unable to determine file type from the file extension, defaulting to js.
  To silence the warning specify a valid type for <...>/node_modules/karma-viewport/dist/adapter/index.js.map in the configuration file.
  See http://karma-runner.github.io/latest/config/files.html
13 10 2020 15:25:18.044:WARN [web-server]: 404: /base/node_modules/karma-viewport/dist/adapter/index.js.map?666b2c333d842e2f4a6850c5c8454da4de299797
13 10 2020 15:25:18.453:WARN [web-server]: 404: /base/node_modules/karma-viewport/dist/adapter/index.js.map?666b2c333d842e2f4a6850c5c8454da4de299797
```

Overall, the combo `included: true, served: false` doesn't seem to make sense:

> `included` - Should the files be included in the browser using `<script>` tag?

Why would we want to load a source map using a script tag?

> `served` - Should the files be served by Karma's webserver?

Yes, we want this file. Why set `served` to `false`?

cc: @altearius

**Comment by squidfunk** [comment]
I'm a little short on time, but if somebody comes up with a PR fixing this, I can issue a bugfix release shortly.

**Comment by squidfunk** [comment]
Fixed by #575.

**Comment by squidfunk** [comment]
Released as part of 1.0.8
